packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -72,3 +72,12 @@ StoryPromoLi.propTypes = {
 StoryPromoLi.defaultProps = {
   border: true,
 };
+
+StoryPromoLiBase.propTypes = {
+  children: node.isRequired,
+  border: bool,
+};
+
+StoryPromoLiBase.defaultProps = {
+  border: true,
+};
```

Comment:
These are identical to `StoryPromoLi`. Can we create a shared `propTypes` and `defaultProps` object?

Reply:
done.